src/samplers/ratelimiting_sampler.js

```diff
@@ -32,7 +32,7 @@ export default class RateLimitingSampler {
         }
 
         this._maxTracesPerSecond = maxTracesPerSecond;
-        this._rateLimiter = new RateLimiter(maxTracesPerSecond);
+        this._rateLimiter = new RateLimiter(maxTracesPerSecond, 2.0);
```

Comment:
This is supposed to not be backwards compatible. Given the new rate limit is 0.001666 (once every 10 mins), if we set maxbalance to maxtracespersecond, we'll never surpass the checkCredit value 1.0. Given that we decide what maxTracesPerSecond is, I don't think this is an issue.